actian_odbc/dialect.tdd

```diff
@@ -882,11 +882,11 @@
     <format-null>
 		<local-type name='date' value='CAST(NULL AS ANSIDATE)' />
 		<local-type name='datetime' value='CAST(NULL AS TIMESTAMP)' />
-		<local-type name='int' value='CAST(NULL AS integer)' />
+		<local-type name='int' value='CAST(NULL AS INTEGER)' />
 		<local-type name='real' value='CAST(NULL AS float)' />
 		<local-type name='str' value='CAST(NULL AS varchar(1))' />
 	</format-null>
     <format-order-by value='Nulls' />    
     <id-allowed-characters value='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_' />
   </sql-format>
-</dialect>
+</dialect>
```

Comment:
Loos like newline at EOF was removed. I'll fix this after merge, no need to fix this instance - something to watch out for with future changes.